website/docs/d/data_lake_pipeline.html.markdown

```diff
@@ -0,0 +1,125 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: data_lake"
+sidebar_current: "docs-mongodbatlas-resource-data-lake"
+description: |-
+    Describe a Data Lake Pipeline.
+---
+
+# Data Source: mongodbatlas_data_lake_pipeline
+
+`mongodbatlas_data_lake_pipeline` describe a Data Lake Pipeline.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+~> **IMPORTANT:** All arguments including the password will be stored in the raw state as plain-text. [Read more about sensitive data in state.](https://www.terraform.io/docs/state/sensitive-data.html)
```

Comment:
> Is this a general statement we are now including?

@Zuhairahmed Could you confirm if we should always include this message? I removed it for now since there is no sensitive information in this resource.

Reply:
Isn't this also dependant on the user's Terraform setup - for example, if they have a remote state via Terraform Cloud, their data won't be stored locally.

More high-level question - what led us to add the disclaimer? If it's a general policy statement, I think we should move it to a central document rather than needing to include it in each example.

Also, happy to make a follow up ticket for our plans here - since this resource doesn't have sensitive information, better to do separately.

Reply:
@colm-quinn good question.   It will always be in plain text but if they are using Terraform Enterprise or Cloud the state is stored centrally and hence more protected.   

But I am note sure when this got added.   I don't remember it so I think @Zuhairahmed this is from after you started?  We have always had messages like this is particularly risky areas, like DB users or API keys, but to Colm's point we could have it just centrally in the main provider doc page and include it on a page only when it's a specifically risky resource.  

Agree a follow up ticket is likely the best course here. Then we can make an intentional decision on this.  @colm-quinn shall we do that and add as a topic for the Prod+Eng meeting?

Reply:
sgtm, let's follow up there!